create-release.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-set -u
+set -xeu
```

Comment:
This is why the release is failing.  We create "current" in case it doesn't exist unconditionally.